server/bleep/Cargo.toml

```diff
@@ -8,8 +8,6 @@ build = "build.rs"
 [features]
 default = []
 debug = ["console-subscriber", "histogram"]
-dynamic-ort = ["ort/use-system"]
-static-ort = ["ort/use-compile", "ort/compile-static"]
```

Comment:
I'd like to keep these for flexibility to keep using our own builds with Nix.

```
[features]
default = ["dynamic-ort"]
debug = ["console-subscriber", "histogram"]
dynamic-ort = ["ort/load-dynamic"]
```

Reply:
Does the app still work on Nix with this PR if `ort/load-dynamic` is disabled?